sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/models/UpdateChatMessageOptions.java

```diff
@@ -35,4 +42,24 @@ public UpdateChatMessageOptions setContent(String content) {
         this.content = content;
         return this;
     }
+
+    /**
+     * Get the metadata property: Message metadata.
+     *
+     * @return the metadata value.
+     */
+    public Map<String, String> getMetadata() {
+        return this.metadata;
+    }
+
+    /**
+     * Set the metadata property: Message metadata.
```

Comment:
Maybe explain the consequence here. What will happen to the metadata? Replace existing set with the new set? Or Update the one with same name but append new ones? Will old ones be removed?

Reply:
This comment/class is coming directly from the REST Swagger and it's the same between all languages in all of our SDKs. The other SDKs and our API also don't clarify whether it overwrites the entire dictionary or just the keys in the update metadata. But I can log a bug on the API to consider updating the comment to be more clear